test/wpt/runner/runner.mjs

```diff
@@ -307,16 +322,23 @@ export class WPTRunner extends EventEmitter {
     }
 
     this.emit('completion')
-    const { completed, failed, success, expectedFailures, skipped } = this.#stats
+
+    const { testsPassed, testsFailed, testsSkipped } = this.#stats
+    console.log(
+      `Test    results for folder [${this.#folderName}]: ` +
+      `completed: ${this.#files.length}, passed: ${testsPassed}, failed: ${testsFailed}, ` +
+      `skipped: ${testsSkipped}`
+    )
+
+    const { subtestsCompleted, subtestsFailed, subtestsPassed, expectedFailures } = this.#stats
```

Comment:
These technically aren't subtests, a test file can contain multiple tests (similar to other frameworks).